test/unit/cli.test.js

```diff
@@ -173,6 +173,36 @@ describe('cli', () => {
       cli.setAssumeYes();
     });
 
+    it('rejects when no default answer is provided for \'input\' type', () => {
+      assert.rejects(() => {
```

Comment:
`assert.rejects` returns a Promise so I think we need to either return it or convert the test to an async function and await (if mocha supports that)

Reply:
whoops, fixed!